### DISCUSSION TITLE
Transition from libmesh to MFEM

### DISCUSSION THREAD

**Discussion by b1llg** [idaholab/moose#28288] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

I watched this presentation: [Multiscale, Multiphysics Simulation Through Application Composition Using MOOSE - Derek Gaston](https://www.youtube.com/watch?v=aHL4FIu_q6k&ab_channel=PETSc). At the end of the presentation, Mr Gaston talks about tansitioning from libmesh to MFEM. I've check all the branches and didn't find any MFEM implementation. 

I wanted to know if it's in development and if it's a closed source version of MOOSE?

**Comment by GiudGiud** [comment]
@friedmud

**Reply by b1llg** [reply]
any update?

**Reply by GiudGiud** [reply]
I dont think this is the plan anymore. 
@lindsayad do we have info we can share for the plan to leverage GPUs?

**Reply by lindsayad** [reply, marked as answer]
We are finalizing our recommendation to DOE about what path we want to choose for GPU acceleration. MFEM is one option. We should have a recommendation within the next few days. However, even once that recommendation has been made, it's far from a done deal. Let's say we do decide to transition to MFEM as a finite element backend, at least for GPU-based computations. It's possible that as we perform the transition we run into some showstopper which we don't yet foresee. Mechanical contact is a big unknown, as an example
